src/role_harvester.js

```diff
@@ -31,7 +31,13 @@ roles.harvester.settings = {
   maxLayoutAmount: 6,
 };
 roles.harvester.updateSettings = function(room) {
-  if (room.storage && room.storage.my && room.storage.store.energy > config.creep.energyFromStorageThreshold && room.energyAvailable > 350 && !room.memory.misplacedSpawn) {
+  if (room.memory.energyStats.average <= 300) {
```

Comment:
I would guess that this should be handled already in the harvester config. I didn't have such cases, but I'm fine to be extra safe.

Reply:
I checked the harvester config and couldn't find where it's handled.

Reply:
Hm, what are you trying to accomplish?

You are checking for `energyStats.average <= 300`, the current code checks for `energyAvailable > 350`. What are the cases where this runs into issues?
Would it make sense to change the `energyAvailable` to `energyStats.average`, so that we don't have a similar special case?!

Reply:
I had the following problem:
- if no room is able to send a nextroomer
- and all creeps are dead
- energyAvailable is higher than 300
no creep has been spawned. This way if the room energyAvailable is below 300 (maybe this should be set higher) there will be at least one low level harvester to get some energy.
I think the problem was, that the `prefixString: 'WMC'` with the normal layout needs too much energy.

Reply:
Hm, `WMC` should be 200 energy, so buildable. Not sure if the creep configurator messed this up.
Can you add exactly this scenario / text as a comment to the if clause, so that we are aware when looking into it again?

Reply:
I will add this as a comment to the if clause.

Reply:
If I'm not wrong the problem seems to be the prefix + layout which is 350 together.

Reply:
K, created an issue: #585